doc/scenario.md

```diff
@@ -281,9 +281,6 @@ List of facts:
   - `instance_dist_dir` - path to instance link to distributed package;
 - `single_instances_for_each_machine` - list of instances (Ansible hosts), one for each physical machine. Can be used,
   for example, in `delegate_to`;
-- `not_expelled_instance` - information about one not expelled instance. It's a dictionary with fields:
```

Comment:
You should add information about this variable in connect_to_membership and set_control_instance steps